### ISSUE TITLE
Have the ColumnDefinition create an IndexDefinition with the same ifNotExists as the TableDefinition?

### ISSUE THREAD

**Issue by Orijhins** [groue/GRDB.swift#838]
### What did you do?

I attempt to create a table with
```swift
database.create(table: Account.databaseTableName, ifNotExists: true) { table in
       table.column("id", .integer).primaryKey(autoincrement: true)
       table.column("accountNo", .integer).indexed()
}
```
to be able to call this piece of code anytime I open a new connection on the database so it doesn't crash when the table already exists

### What did you expect to happen?

Have the .indexed() create an IndexDefinition with the same parameters as the TableDefinition

### What happened instead?

The .indexed() forces most of the parameters, like the ifNotExists to false, thus throwing errors even if the TableDefinition.ifNotExists equals to true

### Environment

**GRDB flavor(s):** GRDB
**GRDB version:** Latest version (5.11-beta)
**Installation method:** CocoaPods
**Xcode version:** 12.0
**Swift version:** 4.2.0
**Platform(s) running GRDB:** macOS
**macOS version running Xcode:** 10.15

**Comment by groue** [comment]
Hello @Orijhins, thank you for the report!

Until a solution is found, you can use the more verbose version:

```swift
if try !database.tableExists(Account.databaseTableName) {
    try database.create(table: Account.databaseTableName) { table in
       table.column("id", .integer).primaryKey(autoincrement: true)
       table.column("accountNo", .integer).indexed()
    }
}
```

**Comment by groue** [comment]
The fix will ship in next version!

**Comment by groue** [comment]
Fixed in GRDB 5.0.0
